tests/functional/metadata/mongodb/getObject.spec.js

```diff
@@ -106,6 +111,25 @@ describe('MongoClientInterface::metadata.getObjectMD', () => {
             });
     }
 
+    function setObjectVFormat(vFormat, cb) {
```

Comment:
should not be needed: just create the bucket with the right format in its metadata?

Reply:
Bucket metadata is stored inside the `BucketInfo` class, and since it doesn't include vFormat we can't really do that